devicon.json

```diff
@@ -3619,7 +3677,7 @@
         ],
         "versions": {
             "svg": [
-		"original",
+        "original",
```

Comment:
Looks like the indentation is wrong here

Reply:
Fixed 🙂